stable/redis/values-production.yaml

```diff
@@ -0,0 +1,230 @@
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
+  accessMode: ReadWriteOnce
+  size: 8Gi
+
+metrics:
+  enabled: true
+  image: oliver006/redis_exporter
+  imagePullPolicy: IfNotPresent
+  imageTag: v0.11
+  # imageCredentials:
+  #   registry: docker.io
+  #   username:
+  #   password:
+  resources: {}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9121"
+
+networkPolicy:
+  ## Enable creation of NetworkPolicy resources.
+  ##
+  enabled: true
+
+  ## The Policy model to apply. When set to false, only pods with the correct
+  ## client label will have network access to the port Redis is listening
+  ## on. When true, Redis will accept connections from any source
+  ## (with the correct destination port).
+  ##
+  allowExternal: true
+
+##
+## Redis Master parameters
+##
+
+## Redis Master additional pod labels
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+podLabels: {}
+
+## Redis Master pod annotations
+podAnnotations: {}
+
+master:
+  ## Use password authentication
+  usePassword: true
+  ## Redis password
+  ## Defaults to a random 10-character alphanumeric string if not set and usePassword is true
+  ## ref: https://github.com/bitnami/bitnami-docker-redis#setting-the-server-password-on-first-run
+  ##
+  port: 6379
+  password: bitnami
+  ## Redis command arguments
+  ##
+  ## Can be used to specify command line arguments, for example:
+  ##
+  ## args:
+  ##  - "redis-server"
+  ##  - "--maxmemory-policy volatile-ttl"
+  args: []
+  ## Redis additional command line flags
+  ##
+  ## Can be used to specify command line flags, for example:
+  ##
+  ## redisExtraFlags:
+  ##  - "--maxmemory-policy volatile-ttl"
+  ##  - "--repl-backlog-size 1024mb"
+  extraFlags: []
+  ## Comma-separated list of Redis commands to disable
+  ##
+  ## Can be used to disable Redis commands for security reasons.
+  ## ref: https://github.com/bitnami/bitnami-docker-redis#disabling-redis-commands
+  ##
+  disableCommands: "FLUSHDB,FLUSHALL"
+
+## Redis Master Node labels and tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+nodeSelector: {"beta.kubernetes.io/arch": "amd64"}
+tolerations: []
+
+## Redis Master Service properties
+serviceType: ClusterIP
+service:
+  annotations: {}
+  loadBalancerIP:
+
+## Redis Master Pod Security Context
+securityContext:
+  enabled: true
+  fsGroup: 1001
+  runAsUser: 1001
+
+## Redis Master resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
```

Comment:
we should not enforce default resource limits/requests.

Reply:
Even in production?